tests/main/apparmor-batch-reload/bin/apparmor_parser.fake

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+debugfile=/tmp/apparmor-parser-calls.debug
+# shellcheck disable=SC2124
+args=$@
+echo "$@" >> $debugfile
```

Comment:
Right. I couldn't think of a better place other than some other hardcoded path that's only writable by root. Since this log file is not super important for the test itself, I changed it to simply log do /dev/kmsg so if it fails, we still have a place to investigate.